src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        for (int i=1; i < 4; i++){
```

Comment:
Так ми відразу зрозуміємо скільки ітерацій циклу відбудеться
```suggestion
        for (int i = 0; i < 3; i++){
```